### PR TITLE
DROID-297: Fix issue where fw update status doesn't trigger in app

### PR DESCRIPTION
**Pull request by nick-sasquatch**

### CODE REVIEW

File: combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt [hunk 1]
```diff
@@ -264,7 +264,6 @@ internal class NetworkManager(
             serialNumber = probe.probeSerialNumber,
             owner = owner,
             settings = settings,
-            dfuConnectedNodeCallback = { },
```
Comment (nick-sasquatch) [inline combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt:267]: This, oddly, wasn't used. I'm guessing we never found a way to integrate the logic with the firmware version coming later.

File: combustion-android-ble/src/main/java/inc/combustion/framework/ble/ProbeManager.kt [hunk 2]
```diff
@@ -622,6 +621,7 @@ internal class ProbeManager(
         _probe.value = _probe.value.copy(
             baseDevice = _probe.value.baseDevice.copy(
                 connectionState = connectionState,
+                fwVersion = fwVersion,
```
Comment (nick-sasquatch) [inline combustion-android-ble/src/main/java/inc/combustion/framework/ble/ProbeManager.kt:624]: It seems like this is the right approach, though if it was omitted in the past, it's not clear why, and the comment reflects the new implementation.

File: combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt (hunk 1 above)
Comment (angrygorilla) [reply combustion-android-ble/src/main/java/inc/combustion/framework/ble/NetworkManager.kt:267]: 🤷🏻

**Review by angrygorilla — APPROVED** [review]
🔥

**Pull request merged**